internal/basicchain/basic.go

```diff
@@ -0,0 +1,244 @@
+package basicchain
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"math/big"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
+	"github.com/nspcc-dev/neo-go/pkg/core/native/nativenames"
+	"github.com/nspcc-dev/neo-go/pkg/core/native/noderoles"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/fixedn"
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/neotest"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/client/nns"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"github.com/stretchr/testify/require"
+)
+
+const neoAmount = 99999000
+
+// Init pushes some predefined set  of transactions into the given chain, it needs a path to
+// the root project directory.
+func Init(t *testing.T, rootpath string, e *neotest.Executor) {
```

Comment:
Then maybe we can move `TestCreateBasicChain` as far?

Reply:
This package is used by test packages, but doesn't have any tests of its own, so it doesn't seem to be appropriate. And we're very much used to `TestCreateBasicChain` being in the `core` (and it does test `core`).